docs/man/git-lfs-config.5.ronn

```diff
@@ -8,6 +8,11 @@ precedence. Most options pertaining to git-lfs are contained in the `[lfs]`
 section, meaning they all named `lfs.foo` or similar, although occasionally an
 lfs option can be scoped inside the configuration for a remote.
 
+These settings are read from a file called `.lfsconfig` found at the root of the
+repository, though legacy support exists to read from `.gitconfig` as well.
+
+The `.lfsconfig` file uses the same format as `.gitconfig`.
```

Comment:
LFS reads configs from a `.lfsconfig` in the root of the working directory, but also overrides any of those settings from `git config -l`. This allows you to override something like `lfs.url` on your local repository if you don't like the setting in `.lfsconfig`.

When I talk about `.gitconfig` support being removed in 2.0, I'm specifically referring to reading `.gitconfig` files in the root of the repository. This is something git does not support anywhere, and was specifically added for Git LFS.

> And should I open a new PR for that or edit on top of this branch?

New PR please, since this is already merged. Thanks :)